requirements.txt

```diff
@@ -1,17 +1,28 @@
 jinja2
```

Comment:
Nit:  We should freeze the rest of these reqs agains their version numbers that are known to work.  Instead of pinning some to latest.  Warehouse is moving people toward pipenv.  Should not block merge.

Reply:
requirements.txt is only used for local build development purposes - setup.py is the cannonical source of requirements. I don't really like that they have to be maintained in separate places.

Still - pinning to versions is not a bad idea

Reply:
I prefer pipenv for this https://github.com/pypa/pipenv